src/createServer.js

```diff
@@ -1,3 +1,54 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable max-len */
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+const createServer = () => {
+  const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const path = new URL(req.url, `http://${req.headers.host}`);
+
+    const originalText = path.pathname.slice(1);
+    const targetCase = path.searchParams.get('toCase');
+
+    const errorMessages = {
+      errors: [],
+    };
+
+    if (!originalText) {
+      errorMessages.errors
+        .push({
+          message: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
```

Comment:
I would put those error messages into some object and then use them like this:
`message: VALIDATION_ERRORS.NO_TEXT`